docs/mocksupport.rst

```diff
@@ -878,3 +878,58 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock InMemory CIM Repository`:
```

Comment:
Make the heading anchor conistent with the actual heading, i.e. 

```
.. _`Mock in-memory CIM repository`:
```

Reply:
DONE